be/src/storage/lake/rowset.cpp

```diff
@@ -97,9 +97,12 @@ StatusOr<ChunkIteratorPtr> Rowset::read(const Schema& schema, const RowsetReadOp
             continue;
         }
 
-        if (options.rowid_range_option != nullptr && !options.rowid_range_option->match_segment(seg_ptr.get())) {
-            continue;
-        }
+        //        if (options.rowid_range_option != nullptr) {
+        //            seg_options.rowid_range_option = options.rowid_range_option->get_segment_rowid_range(this, seg_ptr.get());
+        //            if (seg_options.rowid_range_option == nullptr) {
+        //                continue;
+        //            }
+        //        }
 
```

Comment:
should these comment lines be removed?

Reply:
A TODO comment exists in the header of this method. Rowid range implementation may be added in the future, so it is better to keep the comment as a reference for implementing it later.

```
// TODO: support
//  1. primary key table
//  2. rowid range and short key range
```